test/WorkItemBuiltins/multiple-push-constant.cl

```diff
@@ -0,0 +1,34 @@
+// RUN: clspv -work-dim -global-offset -descriptormap=%t.dmap %s -o %t.spv
+// RUN: spirv-dis -o %t2.spvasm %t.spv
+// RUN: FileCheck %s < %t2.spvasm
+// RUN: FileCheck --check-prefix=DMAP %s < %t.dmap
+// RUN: spirv-val --target-env vulkan1.0 %t.spv
+
+// DMAP: pushconstant,name,dimensions,offset,0,size,4
```

Comment:
You should flip these element to generate:
```
// DMAP: pushconstant,name,global_offset,offset,0,size16
// DMAP: pushconstant,name,dimensions,offset,12,size,4
```

Reply:
Doh! I should have spotted that, thanks.